.github/ISSUE_TEMPLATE/epic-template.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Epic template
+about: template for epics
+title: 'Epic: '
+labels: ''
+assignees: ''
+
+---
```

Comment:
I think we can add `type: epic` here to automatically set the issue type too.

From the [Issue Types Public Preview  discussion](https://github.com/orgs/community/discussions/139933):
> Additionally, you can specify the type qualifier in [issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms) in your repositories, so that each issue created from each template automatically has the type set.

Reply:
Good find! I added three fields:
```
type: epic
projects: ["kiesraad/1"]
area: epic
```

I couldn't test the "type" in my repo outside the Kiesraad organization. The "projects" did not work in my repo. (I am sure it expect the project number.) I don't think the "area" is supported yet, so not sure if it's a good idea to add it already.

Reply:
'Area' is a custom field in our project and I don't think it's possible to set custom fields in the issue template: https://github.com/orgs/community/discussions/63402#discussioncomment-6717854

So maybe it's better to remove that now, because the syntax might be different once it is possible to set custom fields.

Reply:
I removed the 'area' field from the template.